### DISCUSSION TITLE
What's the difference between danfojs and danfojs-node (in NPM)?

### DISCUSSION THREAD

**Discussion by castarco** [opensource9ja/danfojs#232] [Q&A] [answered]
Hi, I was reading about this project, and even before starting I found out that there are two packages that seem almost identical, but with different names, `danfojs` and `danfojs-node`. Is there any difference between them? or Any criteria to choose one or the other?

Thank you.

**Comment by risenW** [comment, marked as answer]
@castarco danfojs is for browser and frameworks like React and Vue, while danfojs-node is for Nodejs environment. 
Each version is optimized for the specific platform.
